### PR TITLE
Various things done (chat buttons/login forms/package installs)

### PR DESCRIPTION
**Pull request by dahvreinhart**
This request encompasses several changes:
   - The 'next' and 'report' buttons have been moved into the chat frame and styled with bootstrap
   - The 'report' button now throws an alert as discussed
   - The login buttons are now being through bootstrap-3 instead of the default Meteor accounts-ui
      - This required some package installs so dont forget to run a 'Meteor npm install' upon pulling

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -5,10 +5,12 @@
     "start": "meteor run"
   },
   "dependencies": {
+    "classnames": "^2.2.5",
```
Comment (madeline-petersen) [inline package.json]: I don't see this being imported anywhere, I think we can remove it using

```
meteor npm uninstall classnames --save
```

Comment (dahvreinhart) [reply package.json]: done

**Comment by madeline-petersen** [comment]
SpaceFrame seems pretty unnecessary now.  What do you think about deleting SpaceFrame, and then renaming SpaceChat ---> SpaceFrame?

File: package.json [hunk 2]
```diff
@@ -9,6 +9,7 @@
     "react": "^15.3.2",
     "react-addons-pure-render-mixin": "^15.3.2",
     "react-dom": "^15.3.2",
+    "react-komposer": "^1.13.1",
```
Comment (madeline-petersen) [inline package.json]: I don't think we're using this either?

```
meteor npm uninstall react-komposer --save
```

Comment (dahvreinhart) [reply package.json]: ok this is done now too

File: imports/ui/SpaceChat/index.js [hunk 3]
```diff
@@ -4,33 +4,38 @@ import Blaze from 'meteor/gadicc:blaze-react-component';
 import { Conversations, spaceGen } from '../../api/ConversationsDbCollection.js';
 import { Mongo } from 'meteor/mongo';
 import { Meteor } from 'meteor/meteor';
-//import {SimpleChats} from '../../api/SimpleChat.js';
+import { Link } from 'react-router'
 
 export default class SpaceChat extends Component {
```
Comment (madeline-petersen) [inline imports/ui/SpaceChat/index.js]: SpaceChat --> SpaceFrame

File: imports/ui/SpaceFrame/index.js [hunk 4]
```diff
@@ -1,17 +1,13 @@
 import React, { Component } from 'react';
```
Comment (madeline-petersen) [inline imports/ui/SpaceFrame/index.js:1]: Delete SpaceFrame folder

Comment (dahvreinhart) [reply imports/ui/SpaceFrame/index.js:1]: finito

**Comment by madeline-petersen** [comment]
LGTM!

**Pull request merged**